roles/sap_ha_install_hana_hsr/tasks/hdbuserstore.yml

```diff
@@ -1,30 +1,20 @@
 ---
-
-# [A] Create and Store Connection Info in hdbuserstore for SYSTEMDB
-- name: SAP HSR - [A] Check if hdbuserstore exists
-  tags: hsr_hdbuserstore
-  become: true
+- name: "SAP HSR - Check if hdbuserstore exists"
   become_user: "{{ sap_ha_install_hana_hsr_sid | lower }}adm"
   shell: |
-        /usr/sap/{{ sap_ha_install_hana_hsr_sid}}/SYS/exe/hdb/hdbuserstore \
-        List {{ sap_ha_install_hana_hsr_hdbuserstore_system_backup_user }} 
+    /usr/sap/{{ sap_ha_install_hana_hsr_sid}}/SYS/exe/hdb/hdbuserstore \
+    List {{ sap_ha_install_hana_hsr_hdbuserstore_system_backup_user }}
   args:
     executable: /bin/bash
   register: sap_ha_install_hana_hsr_hdbuserstore
+  failed_when: false
+  changed_when: false
 
-- name: SAP HSR - [A] Skipping hsr_hdbuserstore
-  tags: hsr_hdbuserstore
-  debug:
-    msg: "Skipping creating {{ sap_ha_install_hana_hsr_hdbuserstore_system_backup_user }} hdbuserstore "
-  when: sap_ha_install_hana_hsr_hdbuserstore.rc==0
-
-- name: SAP HSR - [A] Create and Store Connection Info in hdbuserstore
-  tags: hsr_hdbuserstore
-  become: true
+- name: "SAP HSR - Create and Store Connection Info in hdbuserstore"
   become_user: "{{ sap_ha_install_hana_hsr_sid | lower }}adm"
   shell: |
-        /usr/sap/{{ sap_ha_install_hana_hsr_sid}}/SYS/exe/hdb/hdbuserstore \
-        SET {{ sap_ha_install_hana_hsr_hdbuserstore_system_backup_user }} {{ ansible_hostname }}:3{{ sap_ha_install_hana_hsr_instance_number }}13 SYSTEM '{{ sap_ha_install_hana_hsr_db_system_password }}'
+    /usr/sap/{{ sap_ha_install_hana_hsr_sid}}/SYS/exe/hdb/hdbuserstore \
+    SET {{ sap_ha_install_hana_hsr_hdbuserstore_system_backup_user }} {{ ansible_hostname }}:3{{ sap_ha_install_hana_hsr_instance_number }}13 SYSTEM '{{ sap_ha_install_hana_hsr_db_system_password }}'
```

Comment:
SAP HANA backup is not recommended to perform with SYSTEM user. 

See 

https://help.sap.com/docs/SAP_HANA_PLATFORM/6b94445c94ae495c83a19646e7c3fd56/c4b71703bb571014810ebb38dc59cf51.html

Reply:
Could you please open an issue for follow-up on this?

Reply:
https://github.com/sap-linuxlab/community.sap_install/issues/163